### ISSUE TITLE
@salesforce-ux/slds/enforce-bem-class false positive from Aura merge syntax

### ISSUE THREAD

**Issue by aheber** [salesforce-ux/slds-linter#41]
Aura components allow for code/logic to be embedded into their `html`-like templates. When processed these are often flagged as BEM violations.

```html
<div class="{! 'slds-popover' }">Text</div>
```

Taking the example above I get three errors:
* "The class '{!' does not follow BEM naming convention."
* "The class ''slds-popover'' does not follow BEM naming convention."
* "The class '}' does not follow BEM naming convention."

I'm not sure what to do here. Because this can be mixed text and calculated values it will be hard to correctly evaluate. More than that the logic can compose strings that might result in valid or invalid class names and that is unknowable by only looking at the markup.

```html
<div class="{#'slds-notify_toast slds-theme_' + v.details.variant}" role="alert">
```

There are a lot of variation on this theme so I'm really not sure the right strategy. Maybe checking the value and see if it has either `{!` or `{#` and then ignoring it? Would be unfortunate to skip checking all of these but personally I'd rather miss some things vs. report false positives.

**Comment by kenamverma** [comment]
Hi @aheber, thanks for reaching out. We have published a new version ([0.1.2](https://github.com/salesforce-ux/slds-linter/releases/tag/0.1.2)) for slds-linter which make sure the false positives are not pointed out. Please test and let us know in case of any issues. Thanks.

**Comment by kenamverma** [comment]
We're closing this issue as it's been open for a while without recent activity. If you still have questions or need any help, please don’t hesitate to reach out or open a new issue. Thank you.
